### ISSUE TITLE
[Keyboard Navigation-Embedded-Sign In]: "Sign In" page is not accessible via keyboard.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#10]
Priority: Medium

Repro Steps
1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. “Sign in” page will get appear.
7. Verify that “Sign In” is accessible via keyboard or not.

Expected
“Sign In” page should be accessible via keyboard, as well as swipe gesture.

Actual
“Sign In” page is not accessible via keyboard, while it is accessible swipe gesture.

Note : “Privacy Statement: and “Terms of Use” link is not accessible via swipe gesture.

MAS Rule: MAS4.2.11

Impact
If any control is not accessible with keyboard, So keyboard users will not get the control information in the application.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back
